src/sockets/index.js

```diff
@@ -1,6 +1,6 @@
-'use strict';
+import { socketLogger, worldState } from '../lib/utilities.js';
 
-const { socketLogger: logger, worldState } = require('../lib/utilities');
+const logger = socketLogger;
 
```

Comment:
```suggestion
import { socketLogger as logger, worldState } from '../lib/utilities.js';
```

Reply:
i believe `as` only works on TS. i'd tried it here briefly and it didn't work.

Reply:
i stand corrected! it didn't work for a different reason before. thanks!